src/styles/calendar.scss

```
@@ -0,0 +1,55 @@
+@use './variables' as *;
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+  width: $calendar-flex-width;
+  min-height: $calendar-flex-height;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $days-color;
+    width: $calendar-size;
+    height: $calendar-size;
+    border: $calendar-border solid black;
+    transition: 0.5s ease;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: $calendar-font;
+        font-size: $calendar-font-size;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color;
+      transform: translateY(-$calendar-animation-size);
+    }
+  }
+
+  @each $day in $week-days {
+    &--start-day#{$day} {
```

add here `-` instead of before all of week-days 
`$week-days: '-mon', '-tue', '-wed', '-thu', '-fri', '-sat', '-sun'`
`$week-days: 'mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'`
```suggestion
    &--start-day-#{$day} {
```